traefik/DOCS.md

```diff
@@ -146,6 +146,12 @@ Manually set the DNS servers to use when performing the verification step. Usefu
 
 For more information, see the [Traefik documentation](https://docs.traefik.io/https/acme/#resolvers) regarding this subject.
 
+### Option `pilot_token`
```

Comment:
You've made some great documentation (Thanks!!), but you have not added the `pilot_token` option to the `config.json` file. Please add this and make it an optional string within the schema.

Reply:
I didn't knew this was how to do it. I added the options, I hope it is good.